content/kubermatic/master/tutorials_howtos/CCM_migration/_index.en.md

```diff
@@ -26,11 +26,18 @@ Since the Kubernetes community has planned to deprecate and then remove all the
 controller managers and the Kubernetes documentation explain how to migrate from in-tree to out-of-tree CCM, KKP itself
 needed a mechanism to allow users to migrate their clusters to the out-of-tree implementation, as detailed below.
 
-The supported cloud providers so far are:
-* Hetzner
+The CCM/CSI migration is supported only for the following providers so far:
 * Openstack
 * vSphere
 
+---
+**NOTE**
+
+latest vSphere CSI driver release (v2.3.0) does not support Kubernetes 1.22, hence it is not possible to:
+* Upgrade already migrated vSphere clusters to Kubernetes 1.22.
+* Migrate existing vSphere 1.22 clusters.
+* Create new vSphere clusters with Kubernetes 1.22 (new clusters are by default controller by the external CCM).
+---
```

Comment:
When I said the note block, I meant something like this:

```suggestion
{{% notice note %}}
The latest vSphere CSI driver release (v2.3.0) does not support Kubernetes 1.22, hence it is not possible to:
* Upgrade already migrated vSphere clusters to Kubernetes 1.22.
* Migrate existing vSphere 1.22 clusters.
* Create new vSphere clusters with Kubernetes 1.22 (new clusters are by default controllered by the external CCM).
{{% /notice %}}
```

Please make sure that this renders properly, otherwise, you might have to reformat it a little bit. Also, there's a typo which is fixed in the suggestion.

Reply:
I have had to reformat a bit by collapsing the dotted list to a single sentence because it was not rendered properly.